src/eos-shard-writer-v2.c

```diff
@@ -262,45 +262,80 @@ eos_shard_writer_v2_new_for_fd (int fd)
   return EOS_SHARD_WRITER_V2 (g_object_new (EOS_SHARD_TYPE_WRITER_V2, "fd", fd, NULL));
 }
 
+static int
+compress_blob_to_tmp (GInputStream *file_stream)
+{
+  char tmpl[] = "/tmp/shardXXXXXX";
```

Comment:
Just googled a ton and you're right that it doesn't seem to add a prefix (which I find personally very baffling).